go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/influxdata/telegraf
 
 go 1.22
 
+replace github.com/logzio/azure-monitor-metrics-receiver v1.0.1 => github.com/redbaron/azure-monitor-metrics-receiver v0.0.0-20240521144623-e9f658551b15
```

Comment:
this is https://github.com/logzio/azure-monitor-metrics-receiver/pull/15  , you might want to pull it into your org instead of using my fark.

Reply:
PR was merged and released as 1.0.2 , I have bumped dependency version and removed replacement from go.mod